tests/index.js

```diff
@@ -26,7 +26,7 @@ describe('broccoli-gzip', function(){
     });
 
     builder = new broccoli.Builder(tree);
-    return builder.build().then(function(build) {
+    builder.build().then(function(build) {
```

Comment:
Why aren't these `build` promises being returned anymore? Won't that cause the tests to exit before the build actually completes?